src/app-layout/__integ__/app-layout-focus-delegation.test.ts

```diff
@@ -241,47 +240,6 @@ describe.each(['classic', 'refresh', 'refresh-toolbar'] as const)('%s', theme =>
         );
       });
 
-      test(
-        'drawers focus toggles between open and close buttons',
-        setupTest(
-          async page => {
-            //Altermatomg between triggers because test-1 trigger hidden in overflow menu on mobile,
-            //security has resize button on desktop
-            const triggerSelector = wrapper.findDrawerTriggerById(mobile ? 'security' : 'test-1').toSelector();
```

Comment:
Replaced with a drawer which is always visible and is not resizable ('pro-help').